### DISCUSSION TITLE
adding cdg files

### DISCUSSION THREAD

**Discussion by tonscott** [vicwomg/pikaraoke#226] [Q&A]
Hello, could anyone show me how to copy my cdg files to the sd card on windows pc?

**Comment by dublus** [comment]
Unless you have a Linux box it isn't easy. I bought an addin from Paragon
that adds Linux file systems to Windows. $20, I think.

**Comment by vicwomg** [comment]
Easiest way is to use scp to transfer files to it over the network. There are Windows clients, such as winscp:
https://winscp.net/eng/download.php

You can see the IP of your pikaraoke on the splash screen after it boots up. Use that, and the login / password for your pi. If you used the premade image, its username: pi / password: pikaraoke

**Comment by Romeo1984** [comment]
I usually set up a Samba share and then transfer my 65K CDG files from my Windows machine. It takes about 1.5 hours for me.
